README.md

```diff
@@ -77,4 +77,4 @@ may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
 Unless required by applicable law or agreed to in writing, software distributed
 under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
```

Comment:
please fix - not maintainable 

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Not maintainable

Reply:
looks good, readable

---
**Sema Reaction:** :ok_hand: This code looks good | **Sema Tags:** Readable

Reply:
write something here

---
**Sema Reaction:** :trophy: This code is awesome | **Sema Tags:** Inelegant, Not reusable, Brittle, Not maintainable

Reply:
The product creates a search index of private or sensitive documents, but it does not properly limit index access to actors who are authorized to see the original information.
The software does not properly clean up and remove temporary or supporting resources after they have been used.

Reply:
The product does not clean up its state or incorrectly cleans up its state when an exception is thrown, leading to unexpected state or control flow.
The product does not clean up its state or incorrectly cleans up its state when an exception is thrown, leading to unexpected state or control flow.

---
**Sema Tags:** Not secure